concourse-chrome-driver/terraform/deployments/670214072732/module-codebuild-build-container-docker-hub.tf

```diff
@@ -0,0 +1,27 @@
+module "codebuild-build-container-docker-hub-cd" {
+  source              = "github.com/alphagov/cyber-security-shared-terraform-modules//codebuild/codebuild_build_container_docker_hub"
+  stage_name          = "BuildDockerImages"
+  action_name         = "BuildAndPushCdImage"
+  service_role_name   = var.codebuild_service_role_name
+  docker_hub_repo     = "gdscyber/cd-chrome-driver"
+  build_context       = var.build_context
+  dockerfile          = var.dockerfile
+  docker_hub_username = local.docker_hub_username
+  docker_hub_password = local.docker_hub_password
+  pipeline_name       = var.pipeline_name
+  environment         = var.environment
+}
+
+module "codebuild-build-container-docker-hub-concourse" {
+  source              = "github.com/alphagov/cyber-security-shared-terraform-modules//codebuild/codebuild_build_container_docker_hub"
+  service_role_name   = var.codebuild_service_role_name
+  docker_hub_repo     = "gdscyber/concourse-chrome-driver"
+  stage_name          = "ConcourseDockerImage"
```

Comment:
This stage should be BuildDockerImages - It's the stage name from the codepipeline

Reply:
Updated

Reply:
It's still showing as "ConcourseDockerImage" here - have you pushed the change?

Reply:
Applied the TF, just testing it before I push the commits.